demos/form-control/dropdown-plus.php

```diff
@@ -21,7 +21,7 @@
 
 $form->addControl('category_id', [Form\Control\Dropdown::class, 'model' => new Category($app->db)]);
 $form->addControl('sub_category_id', [Form\Control\DropdownCascade::class, 'cascadeFrom' => 'category_id', 'reference' => Category::hinting()->fieldName()->SubCategories]);
-// DEBUG only, uncomment before merge!!! $form->addControl('product_id', [Form\Control\DropdownCascade::class, 'cascadeFrom' => 'sub_category_id', 'reference' => SubCategory::hinting()->fieldName()->Products]);
+$form->addControl('product_id', [Form\Control\DropdownCascade::class, 'cascadeFrom' => 'sub_category_id', 'reference' => SubCategory::hinting()->fieldName()->Products]);
```

Comment:
when this line is commented, tests pass

@ibelar, please help

Reply:
when this line is commented, tests pass

@georgehristov can you please check why?